src/utils/getAllContribsUtility.js

```diff
@@ -0,0 +1,27 @@
+// Fetches all the publiclab's repositories
+function getAllRepos(org) {
+
+  // This array is used to store all the repositories fetched from Github
+  let repos = [];
+
+  return fetch('https://api.github.com/users/publiclab/repos?sort=pushed&direction=desc&per_page=100')
```

Comment:
Here, shouldn't we point it at `org` though we can default to `publiclab`?

Reply:
On it.